Documentation/profiles.md

```diff
@@ -53,7 +53,17 @@ TODO
 
 ### Docker with User Namespaces
 
-TODO
+This profile allows user to start container with support of user namespace. The
+user on the host is `dockremap` which is automatically created by the docker
+daemon. All the containers started with user namespace enabled will have same
+`UID`, with following mapping:
+
+```
+dockremap:100000:65536
```

Comment:
This is the syntax from `/etc/subuid` (`man subuid`).

But when setting up the mapping, docker will just write `0 100000 65536` into uid_map (syntax from `/proc/$$/uid_map`, from `man user_namespaces`) and the kernel will not know about the link with user `dockremap` on the host (that's just a convention for the `newuidmap` binary with either setuid or file capabilities):
```
$ getcap /usr/bin/newuidmap
/usr/bin/newuidmap = cap_setuid+ep
```

So I would suggest to use the uid_map notation in the documentation, since that's something that is part of the Linux kernel API.